content/anchor-pdas.md

```diff
@@ -361,7 +361,7 @@ Within the instruction logic, we’ll populate the data of the new `movie_review
 
 ```rust
 #[program]
-pub mod movie_review{
+pub mod anchor_movie_review_program{
```

Comment:
Why rename the program? `anchor` is implied and we know its a program because we just told Rust it was a Solana program. We can probably do without this change unless there's a very good reason.